src/librustc_mir/build/expr/as_rvalue.rs

```diff
@@ -381,28 +377,24 @@ impl<'a, 'gcx, 'tcx> Builder<'a, 'gcx, 'tcx> {
 
     // Helper to get a `-1` value of the appropriate type
     fn neg_1_literal(&mut self, span: Span, ty: Ty<'tcx>) -> Operand<'tcx> {
+        let ty = self.hir.tcx().lift_to_global(&ty).unwrap();
```

Comment:
I'm not too happy with all the `lift_to_global` calls. Can we make the lifetime annotations on `from_bits` and friends work out so we don't need that anymore?

Reply:
We need global types here since we must compute their layout (which requires `Ty<'gcx>`).

Reply:
The `lift_to_global` calls have been removed.